R/check.R

```diff
@@ -56,14 +58,17 @@ setMethod(f = "check",signature(.Object = "geojob"), definition = function(.Obje
 		status <- sapply(xmlChildren(root[["Status"]]),xmlValue)
 		process$status	<-	status[[1]]
 		process$statusType <- sapply(xmlChildren(root[["Status"]]),xmlName)[[1]]
-		
 		if (process$status == "Process successful"){
 			root <- xmlRoot(checkForCompleteResponse)
+			process$percentComplete <- "100"
 			process$URL <- as.character(xpathApply(root, "//@href", namespaces = checkResponseNS)[[1]])
 		} else if (process$status == ""){
 		  process$status <- "ProcessStarted"
 		} else if (substr(process$status, 1, 34) == "org.n52.wps.server.ExceptionReport"){
 		  process$status <- "ProcessFailed"
+		} else if (process$status == "Process Started") {
+		  process$percentCompleted <- as.character(xpathApply(root, "//wps:ProcessStarted", 
```

Comment:
I think you could avoid xmlGetAttr by doing `//wps:ProcessStarted/@percentCompleted` like the href is extracted above, but probably not necessary.